### DISCUSSION TITLE
App not working

### DISCUSSION THREAD

**Discussion by Jayantswami321** [JunkFood02/Seal#1737] [Q&A]
App version: 1.13.0 (11302)
Device information: Android 11 (API 30)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.09.27

URL: https://www.instagram.com/reel/DAloX6IOnKF/?igsh=MWRqNGJpZWtrdzg4eA==
Deprecated Feature: Support for Python version 3.8 has been deprecated. Please update to Python 3.9 or above
WARNING: [Instagram] DAloX6IOnKF: General metadata extraction failed (some metadata might be missing).
WARNING: [Instagram] unable to extract shared data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [Instagram] Main webpage is locked behind the login page. Retrying with embed webpage (some metadata might be missing).
WARNING: [Instagram] unable to extract additional data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [Instagram] DAloX6IOnKF: Requested content is not available, rate-limit reached or login required. Use --cookies, --cookies-from-browser, --username and --password, --netrc-cmd, or --netrc (instagram) to provide account credentials

This is the problem what should i do

**Comment by Deepesh9620** [comment]
![Screenshot_20241007-220228_Seal](https://github.com/user-attachments/assets/cbd699e1-5c15-443c-bf7b-c0d5dc933de2)
I get the same error

**Reply by Shiroha1768** [reply]
Need to get cookies![Screenshot_2024-10-18-11-59-36-23_2b0c70832456616f1f794bb05bb8d2fa.jpg](https://github.com/user-attachments/assets/bb1c135f-4455-4066-b2f9-bd98a7f98d5b)
